GetCertificate/printer.ts

```diff
@@ -203,14 +203,23 @@ export const printExpiredOrRevokedInfo = (
   lang: o.Option<PreferredLanguage>
 ): string => getPrinterForLanguage(lang).expiredInfoPrinter();
 
+/**
+ * Check if Vaccination is a Booster
+ *
+ * @param v the VaccinationEntry to check
+ * @returns true if is booster, false otherwise
+ */
+export const isBooster = (v: VaccinationEntry): boolean =>
+  v.dn === 3 || (v.dn === 2 && v.sd === 1);
+
 /**
  * Check if Vaccination has ended
  *
  * @param v the VaccinationEntry to check
  * @returns true if process ended, false otherwise
  */
 export const isVaccinationProcessEnded = (v: VaccinationEntry): boolean =>
-  v.dn >= v.sd; // we may have more doses than the total in case of additional, "booster doses"
+  v.dn === v.sd;
 
```

Comment:
Aren't those 2 condition wrongly overlapped? How could a isBooster==true certificate be isVaccinationProcessEnded==false for the case 2/1 ?

Reply:
We are using them in a ternary operator, first checking the booster dose and only after checking the end of the process. 
So I though to separate the booster from the simple end of vaccination process.
Indeed we have the case 3/3 that fits both, so there is no value on this.. we can leave as it is, sure.